defaults/main.yml

```diff
@@ -29,8 +29,8 @@ github_api_url: "https://api.github.com"
 # Personal Access Token for your GitHub account
 access_token: "{{ lookup('env', 'PERSONAL_ACCESS_TOKEN') }}"
 
-# Is it the runner for organization or not?
-runner_org: no
+# Is it the runner for a "repo", an "org" organization or an "ent" enterprise?
```

Comment:
Let's use the full names, `repository`, `organisation`, `enterprise`. There's no benefit to these abbreviations and they they add the to opportunity for misunderstanding.

Reply:
Fair point, I've fixed this up in the latest commit.